py/sight/demo/portfolio_demo.py

```diff
@@ -152,13 +147,13 @@ def main_wrapper(argv):
       print('optimizer_config : ', optimizer_config)
 
       # Configure decision and launch trials
-      opt_obj, decision_configuration = decision.configure_decision(
+      decision_configuration = decision.configure_decision(
           sight, question_label, question_config, optimizer_config,
           optimizer_type)
       trials.launch(decision_configuration, sight)
 
       # Start worker jobs
-      start_worker_jobs(sight, optimizer_config, workers_config, optimizer_type)
+      trials.start_worker_jobs(sight, optimizer_config, workers_config, optimizer_type)
```

Comment:
Should we remove these calls  since we're calling it inside of Sight? (also, start_worker_jobs takes in the question_label)